packages/ptokens-assets-evm/test/ptokens-evm-asset.test.ts

```diff
@@ -30,6 +30,8 @@ describe('EVM asset', () => {
         },
         factoryAddress: 'factory-address',
         hubAddress: 'hub-address',
+        pTokenAddress: '0x6a57e6046405eb1a075c3ea51de6447171417e24',
```

Comment:
either leave a fake ptokens address, e.g. `ptoken-address`, or change hubAddress etc with real values

Reply:
not implemented yet, new because of new use of type 0x${string} as required by viem more widely.
Many tests has 0x type of string because of this